### DISCUSSION TITLE
Service Mesh

### DISCUSSION THREAD

**Discussion by sreejesh123** [strimzi/strimzi-kafka-operator#5356] [Q&A]
Hi, we want to integrate Strimzi with Service Mesh (istio or ASM)  which will help with mainly  company wide cert management system , any experience with this kind of integration and anything to look for?

**Comment by scholzj** [comment]
There is currently no support for integrating with Service Mesh. As far as I know, you can have Kafka running outside of the mesh and configure Istio to let the Kafka clients running inside it to connect to the outside. But you cannot run Strimzi inside the mesh.
